### PR TITLE
Incogurity  Class and cleaninng data from punc 

### PR DESCRIPTION
**Pull request by amir4859**
Incongurity Vecrorizer and cleaning. data

### CODE REVIEW

File: DataLoaderFromFile.py [hunk 1]
```diff
@@ -22,5 +26,56 @@ def break_each_sentence_into_tokens(self):
...
             for word in words:
-                new_sentence.append(word)
+                # for comma w/o spaces
+                fixed_word = self.syntax_fixer(word)
+                new_sentence += fixed_word
             self.all_sentences_splitted_by_word.append(new_sentence)
+
+    def syntax_fixer(self, word):
+        """
+        The function check for irrelevant punctuation and split/ remove etc
```
Comment (artik00) [inline DataLoaderFromFile.py]: The function checks

File: IncogurityVectorizer.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+# a = model.similarityilarity('in', 'for')
+
+class IncogurityVectorizer:
+
+    def __init__(self, sentences, similarity_func):
+        self.similarity_func = similarity_func
+        self.sentences = sentences
+        self.vector = []
+        for sent in sentences:
```
Comment (artik00) [inline IncogurityVectorizer.py]: sent=sentence

File: IncogurityVectorizer.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+            for pair in pairs:
+                try:
+                    similarity_score = self.similarity_func(pair[0], pair[1])
+                    similarity_scores.append(similarity_score)
+                except KeyError as e:
+                    print(f"IncogurityVectorizer - word wasnt found {str(e)}, adding 0 as similarity score for {pair}")
+                    similarity_scores.append(0)
+            max_similarity = max(similarity_scores)
+            min_similarity = max(similarity_scores)
```
Comment (artik00) [inline IncogurityVectorizer.py]: you probably meant min here

Comment (amir4859) [reply IncogurityVectorizer.py]: done

File: DataLoaderFromFile.py (hunk 1 above)
Comment (amir4859) [reply DataLoaderFromFile.py]: done

**Pull request merged**